src/webgl/light.js

```diff
@@ -9,68 +9,82 @@ import p5 from '../core/main';
 import * as constants from '../core/constants';
 
 /**
- * Creates an ambient light with a color. Ambient light is light that comes from everywhere on the canvas.
- * It has no particular source.
+ * Creates an ambient light with the given color.
+ *
+ * Ambient light does not come from a specific direction.
+ * Objects are evenly lit from all sides. Ambient lights are
+ * almost always used in combination with other types of lights.
+ *
+ * Note: lights need to be called (whether directly or indirectly)
+ * within draw() to remain persistent in a looping program.
+ * Placing them in setup() will cause them to only have an effect
+ * the first time through the loop.
+ *
  * @method ambientLight
- * @param  {Number}        v1      red or hue value relative to
- *                                 the current color range
- * @param  {Number}        v2      green or saturation value
- *                                 relative to the current color range
- * @param  {Number}        v3      blue or brightness value
- *                                 relative to the current color range
- * @param  {Number}        [alpha] the alpha value
+ * @param  {Number}        v1       red or hue value relative to
+ *                                   the current color range
+ * @param  {Number}        v2       green or saturation value
+ *                                   relative to the current color range
+ * @param  {Number}        v3       blue or brightness value
+ *                                   relative to the current color range
+ * @param  {Number}        [alpha]  alpha value relative to current
+ *                                   color range (default is 0-255)
  * @chainable
  *
  * @example
  * <div>
  * <code>
  * createCanvas(100, 100, WEBGL);
- * ambientLight(0);
- * ambientMaterial(250);
+ * ambientLight(0); // black light (no light)
+ * ambientMaterial(255, 127, 80); // coral material
  * sphere(40);
  * </code>
  * </div>
+ * @alt
+ * sphere with coral color under black light
+ *
+ * @example
  * <div>
  * <code>
  * function setup() {
  *   createCanvas(100, 100, WEBGL);
  * }
  * function draw() {
  *   background(51);
- *   ambientLight(100); // white light
- *   ambientMaterial(255, 102, 94); // magenta material
+ *   ambientLight(255); // white light
```

Comment:
I would try to keep as many visual elements similar between the block ambient light example and the white example (ie same shape, same background color).

Reply:
Good point. Updated both examples to be identical except for the single light color difference.